diagnostics/include/protocol.h

```diff
@@ -95,7 +95,7 @@ inline std::string run(const std::vector<std::string>& tokens, Registrar& regist
         }
         return usage();
       }
-      if (command == "list") {
+      if (command == "list-keys") {
```

Comment:
My preference would be to change `list-keys` in status to `list`, rather than the other way around.. I think the shorter name is better for a cli.

Reply:
sure, agreed, changed the command to `list` for both `status` and `perf` commands